services/api-server/src/simcore_service_api_server/models/schemas/jobs.py

```diff
@@ -252,14 +239,10 @@ class Config(BaseConfig):
         schema_extra = {
             "example": {
                 "job_id": "145beae4-a3a8-4fde-adbb-4e8257c2c083",
-                "state": TaskStates.STARTED,
+                "state": RunningState.STARTED,
                 "progress": 3,
                 "submitted_at": "2021-04-01 07:15:54.631007",
```

Comment:
Can you please add the new definitions of submitted_at, started_at and stopped_at  (as written in the PR description) as descriptions to these fields int he `Job` model?